src/components/MainHeader/MainHeaderStyles.ts

```diff
@@ -11,6 +11,13 @@ const StyledMainHeader = styled.header`
   justify-content: space-between;
   padding: 0 1rem;
   background-color: #fff;
```

Comment:
it would be better and convenient to use only the colors defined in your app styles  --color-main is your white.